internal/place/client.go

```diff
@@ -141,6 +141,10 @@ func (a Address) String() string {
 	return strings.Join(a.Lines(), ", ")
 }
 
+func (a Address) HTML() string {
```

Comment:
I see why you've added this, but I think it should return `template.HTML` so you don't need the `html` fn

Reply:
or alternatively we could have an `address-summary-row` template

Reply:
I'd originally had `template.HTML` as the return type but I think there was a constraint around where I was using it...I like the idea of splitting into its own template rather than trying to cram it into one 👍